source/_posts/2024-05-22-companion-app-for-ios-20245-assist.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "Companion app for iOS 2024.5: Let me Assist you 🍎"
+description: "Assist on iOS and companion App progress so far."
+date: 2024-05-22 00:00:02
+date_formatted: "May 22, 2024"
+author: Bruno Pantaleão
+comments: true
+categories: iOS Release-Notes
+og_image: /images/blog/2024-05-ios-assist/ios-og.jpg
+---
+
+Hey, this is Bruno. It has been a great few months since I  joined Nabu Casa to work full-time on the Home Assistant iOS app, again a big thanks to our Home Assistant Cloud subscribers for making this possible. Today, I would like to recap what has been introduced since [the last blog post for version 2024.1](https://www.home-assistant.io/blog/2024/01/29/companion-app-for-ios-20241-carplay/) and discuss more about how Assist is being integrated into the Apple ecosystem - along with a number of other improvements to the app.
```

Comment:
```suggestion
Hey, this is Bruno. It has been a great few months since I joined Nabu Casa to work full-time on the Home Assistant iOS app, again a big thanks to our Home Assistant Cloud subscribers for making this possible. Today, I would like to recap what has been introduced since [the last blog post for version 2024.1](/blog/2024/01/29/companion-app-for-ios-20241-carplay/) and discuss more about how Assist is being integrated into the Apple ecosystem - along with a number of other improvements to the app.
```